app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml

```diff
@@ -55,6 +55,7 @@
 
         <TextView
           android:id="@+id/add_profile_text"
+          style="@style/TextViewCenter"
```

Comment:
This isn't being centered in the layout--is this correct?

Reply:
No, fixed it.